src/modelmanager.cpp

```diff
@@ -585,10 +585,18 @@ Status ModelManager::loadCustomLoadersConfig(rapidjson::Document& configJson) {
 
 Status ModelManager::loadMetricsConfig(rapidjson::Document& configJson) {
     const auto itr2 = configJson.FindMember("monitoring");
+    auto& config = ovms::Config::instance();
     if (itr2 == configJson.MemberEnd() || !itr2->value.IsObject()) {
+        if (config.metricsEnabled()) {
```

Comment:
metricsList should be included, it also can not be used when the config.json has metrics settings

Reply:
Does it matter, since using metric_list without metrics_enabled yields following error "metrics_enable setting is missing, required when metrics_list is provided"?